rideshare.rb

```diff
@@ -0,0 +1,121 @@
+drivers = [
+  {
+    driver_ID: 'DR0001',
+    rides: [
+      {
+        rider_ID: 'RD0003',
+        date: '2/3/2016',
+        cost: '10',
```

Comment:
It's not 'wrong', but you could save a lot of vertical space by having the hash elements on a single line.